test/bdd/features/vc_e2e_api.feature

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+@all
+@vc_rest
+Feature: Using VC REST API
+
+  @create_profile
+  Scenario: Create a profile.
+    Then  Send request to create a profile with profile request "${PROFILE_REQUEST1}" and receive the profile response "${EXPECTED_PROFILE_RESPONSE1}"
+
+  @get_profile
+  Scenario: Get a previously stored profile.
+    Given Send request to create a profile with profile request "${PROFILE_REQUEST2}" and receive the profile response "${EXPECTED_PROFILE_RESPONSE2}"
+    Then  Send request to get a profile with id "profile2" and receive the profile response "${EXPECTED_PROFILE_RESPONSE2}"
+
+  @create_credential
+  Scenario: Create a credential under a previously created profile.
+  Given Send request to create a profile with profile request "${PROFILE_REQUEST3}" and receive the profile response "${EXPECTED_PROFILE_RESPONSE3}"
+  Then  Send request to create a credential with credential request "${CREDENTIAL_REQUEST1}" and receive a verified credential with issuer ID "did:peer:22" and issuer name "profile3"
+
+  @store_credential
+  Scenario: Store a credential after creating it.
+    Given Send request to create a profile with profile request "${PROFILE_REQUEST4}" and receive the profile response "${EXPECTED_PROFILE_RESPONSE4}"
+    Given Send request to create a credential with credential request "${CREDENTIAL_REQUEST2}" and receive a verified credential with issuer ID "did:peer:22" and issuer name "profile4"
+    Then  Send request to store a credential with StoreVCRequest "${STORE_VC_REQUEST1}"
+
+  @retrieve_credential
```

Comment:
We probably don't need more than one overall test.
1) Create and retrieve profile
2) Create and retrieve credential
3) Verify credential